packages/bot-web-ui/src/components/dashboard/tour-slider.tsx

```diff
@@ -67,6 +67,7 @@ const TourSlider = observer(() => {
     const [slider_content, setContent] = React.useState<string | string[]>('');
     const [slider_header, setheader] = React.useState<string>('');
     const [slider_image, setimg] = React.useState<string>('');
+    const [slider_media, setmedia] = React.useState<string>('');
```

Comment:
would be better camel case according to syntax agreements in wiki.js

Reply:
@maryia-matskevich-deriv yes ofcourse. Done ✅